rust-toolchain.toml

```diff
@@ -1,2 +1,2 @@
 [toolchain]
-channel = "nightly-2023-07-01"
+channel = "stable-2023-08-03"
```

Comment:
Don't change this now all the others features are not removed yet

Reply:
done